### ISSUE TITLE
Pretrained mobilenet v1 imagenet

### ISSUE THREAD

**Issue by selkerdawy** [hyang1990/energy_constrained_compression#1]
Hello,

Thanks for sharing the code, I was wondering if you can share your pretrained weights for mobilenet imagenet for comparison?

Edit: I was able to use the models uploaded in [here](https://github.com/ruotianluo/pytorch-mobilenet-from-tf) after renaming the weights achieving 71.2% accuracy.

For those interested in the [weights](https://drive.google.com/open?id=1WMZA1TVVsPv1UCOxehi01bhYXGAyIEbl)

Best,
Sara
